src/pages/how-to-be-sponsors.tsx

```diff
@@ -4,10 +4,51 @@ import "bootstrap/dist/css/bootstrap.min.css";
 
 import BasePage from "../components/BasePage/BasePage";
 
+const title = "Become a Sponsor"
+
+const style = {
+  "width": "50%",
+  "margin-left": "auto",
+  "margin-right": "auto",
+}
+
+const bronze = {
+  "color": "#cd7f32",
+  "font-weight": "bold"
+};
+
+const silver = {
+  "color": "#c0c0c0",
+  "font-weight": "bold"
+};
+
+const gold = {
+  "color": "#ffd700",
+  "font-weight": "bold"
+};
+
+const pink = {
+  "color": "#FF007f",
+  "font-weight": "bold"
+};
+
 const page = () => (
-  <BasePage pageName="How to be a sponsor">Welcome to the How to be a sponsors page!</BasePage>
+  <BasePage pageName={title}>
+    <div style={style}>
+      <p>Do you own, work for, or know a business that would like to support our program and receive advertisement opportunities in return?</p>
+      <p>
+        We're always seeking funding and sponsors to support our budget, which was grown to nearly $20,000 per year.
+        We offer four tiers of sponsorship: <span style={bronze}>Bronze</span>, <span style={silver}>Silver</span>, <span style={gold}>Gold</span>, and <span style={pink}>Pink</span>. Each offers logo placement with increasing visibility, starting with our website for Bronze and culminating in a large logo on our competition robot.
+        See our sponsorship packet below for details!
+      </p>
+      <p>Contact Sam Goldman at <a href="mailto:sgoldman@cherrycreekschools.org?subject=BlazerBots Sponsorship Inquiry">sgoldman@cherrycreekschools.org</a> with any questions or to get started.</p>
+
+      <p>We look forward to working with you and to seeing your logo on our robot!</p>
+    </div>
+    <iframe src="https://drive.google.com/file/d/1zJ_RThOwgtSONbEUVSGEswerpjsNMbjc/preview" width="640" height="480"></iframe>
```

Comment:
I've played around with this and have some thoughts:
 - In order to get this looking good on desktop and mobile the width needs to be dynamic (ex `100%` rather than pixels)
 - A height looked fine dynamic or static if it was large enough. I think having the height be exactly 1 page of the PDF would be great, in which case it needs to be dynamic.
 - To achieve a height of ~1 PDF page and a dynamic width, we can achieve that by knowing there exists a fixed ratio for page size, and use a CSS property that will couple height to width.

The following snippet gave good results in my opinion:
```tsx
const sponsorPacketUrl = "https://drive.google.com/file/d/1zJ_RThOwgtSONbEUVSGEswerpjsNMbjc/preview";

/* other style code here */

const pdfContainerStyle = {
  "aspect-ratio": "8.5/11",
  width: "80%",
  margin: "auto",
}
```

```tsx
<iframe src={sponsorPacketUrl} style={pdfContainerStyle}/>
```

Desktop:
![image](https://github.com/OHSBlazerbots/blazerbots/assets/6364480/1e12ede9-241a-4646-aa69-208d11dc8312)

Mobile:
![image](https://github.com/OHSBlazerbots/blazerbots/assets/6364480/51de9c5e-7f7a-4c85-afa3-15ddad389bb5)

Note: The rendered PDF is not 1-page tall (by default) if you use a large screen (ie `iframe` is bigger than 8.5x11in). In that case, the default viewer zoom settings put padding around the page width. I don't think that is a problem for us, I just want to be clear this approach doesn't address that.
![image](https://github.com/OHSBlazerbots/blazerbots/assets/6364480/c3818e28-f546-400d-909a-7e4613f49755)